addon/mixins/model-records/columns.js

```diff
@@ -10,9 +10,12 @@ function columnContains(columnType, parameter) {
 
 export default Ember.Mixin.create(RecordTypeMixin, {
   columns: Ember.computed('model', function() {
-    var klass = this.get('container').lookup('data-adapter:main').getModelTypes().findBy('name', this.get('recordType')).klass;
+    const adapter = this.get('container').lookup('data-adapter:main');
```

Comment:
We prefer to use the `get(this, 'container')` style rather than calling `get` off the receiver.